code/datums/shuttles.dm

```diff
@@ -216,11 +216,16 @@
 /datum/map_template/shuttle/emergency/meteor
 	suffix = "meteor"
 	name = "Asteroid With Engines Strapped To It"
-	description = "A hollowed out asteroid with engines strapped to it, the hollowing procedure makes it very difficult to hijack but is very expensive. Due to its size and difficulty in steering it, this shuttle may damage the docking area."
+	description = "A hollowed out asteroid with engines strapped to it. Due to its size and difficulty in steering it, this shuttle may damage the docking area. Only available after a meteor wave, for both financial and practicality reasons."
```

Comment:
can you come up with some better lore for this, even what you said in the PR "CentComm does not believe in letting a good meteor wave go to waste."

Reply:
wadasdasd